thirdeye/thirdeye-frontend/app/mirage/endpoints/rootcause.js

```diff
@@ -45,4 +45,28 @@ export default function(server) {
   server.get('/data/maxDataTime/metricId/1', () => {
     return 1517846399998;
   });
+
+  /**
+   * Retrieves information about a session
+   */
+  server.get('/session/1', () => {
+    return {
+      "id" : 1,
```

Comment:
QQ: do we need quotes around the key names here? 
I understand that we're trying to replicate a JSON response, but I feel this is already handled by mirage.

Reply:
No, there's no need for quotes. I'll remove them.